internal/jobs/launch_instance_aws.go

```diff
@@ -42,8 +43,13 @@ type LaunchInstanceAWSTaskArgs struct {
 	ARN *clients.Authentication
 }
 
-// Unmarshall arguments and handle error
+// HandleLaunchInstanceAWS unmarshalls arguments and handles error
```

Comment:
Just so that Golang's comment formatting is happy. Comment above a function should start with the function name. Not a big issue, but since I was already refactoring...